lib/screens/ingredients/ingredients_page.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+
+class IngredientsPage extends StatelessWidget {
+  const IngredientsPage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Ingredients'),
+      ),
+      body: const Center(
+        child: Text('Ingredients'),
```

Comment:
- [ ] Let's add this raw Strings in the Internationalization standard for future languages implementations